frontend/packages/ux-editor/src/components/config/RuleComponent.module.css

```diff
@@ -0,0 +1,109 @@
+.modalHeader {
```

Comment:
I think we should add these CSS variables on top of this file to ensure that the variables exist.
```css
  --primary-color: #008fd6;
  --primary-color-700: #022f51;
  --success-color: #17c96b;
  --danger-color: #e23b53;
  --text-color: #000;
  --paper-color: #fff;
  --disabled-color: #e9ecef;
  --overlay-gradient: rgba(30, 174, 247, 0.3);
```